### ISSUE TITLE
Feature request: Support for shortcodes

### ISSUE THREAD

**Issue by yakovsh** [Knawat/featured-image-by-url#44]
Would you consider adding support for shortcodes within the URL and alt text? Should be fairly simple to do via "do_shortcode()":
https://developer.wordpress.org/reference/functions/do_shortcode/
